src/main/java/com/minecolonies/coremod/entity/citizen/citizenhandlers/MovementHandler.java

```diff
@@ -119,4 +119,11 @@ else if (this.action == net.minecraft.entity.ai.controller.MovementController.Ac
             this.mob.setMoveForward(0.0F);
         }
     }
+
+    @Override
+    public void setMoveTo(double x, double y, double z, double speedIn)
+    {
+        super.setMoveTo(x, y, z, speedIn);
+        this.action = MovementController.Action.MOVE_TO;
+    }
```

Comment:
No other side-effects of this?

Reply:
not as far as I could see, tried jumping down/walking up/down mountains etc seemed just fine